examples/simple/.envrc

```diff
@@ -0,0 +1,7 @@
+source_url \
+  https://raw.githubusercontent.com/juspay/omnix/6132d1054e824cc63c05c7b1344fddbd1f4ad472/omnixrc \
```

Comment:
Aside (for omnix): I feel like we should probably host `https://omnix.page/om/develop/omnixrc/4` (where 4 is version) to give users shorter URLs.